src/main/java/blue/lapis/pore/impl/PoreServer.java

```diff
@@ -767,4 +775,65 @@ public void sendPluginMessage(Plugin source, String channel, byte[] message) {
 
         return result;
     }
+    
+    
+    @SuppressWarnings("unchecked")
+    public void registerCommands() {
+        org.bukkit.plugin.Plugin[] bPlugins = this.getPluginManager().getPlugins();
+
+        for(org.bukkit.plugin.Plugin bPlugin : bPlugins) {
+        	if(bPlugin.getDescription().getCommands() != null) {
+        		getLogger().info(String.format("Loading commands for %s", bPlugin.getDescription().getName()));
+
+        		Iterator<Entry<String, Map<String, Object>>> mapi = bPlugin.getDescription().getCommands().entrySet().iterator();
+
+        		while(mapi.hasNext()) {
```

Comment:
The Lapis coding style calls for a space between keywords like this and parentheses. It's not a big deal in this case, as I can fix it while merging, but keep this in mind in the future. :)